README.md

```diff
@@ -134,77 +138,130 @@ ROOT
+|Test|Info,Warning|Registry entries|Provides information about the number of registry entries found in loependeJournal.xml and tests whether it matches the number of registry entries found in arkivstruktur.xml.|
+|Test|Info,Warning|Period containment of registry entries|Tests whether the created and finalized dates of all registry entries in loependeJournal.xml are within the archival period specified in arkivuttrekk.xml, and whether the same registry entries (same systemIDs) can be found in arkivstruktur.xml.|
 
 **offentligjournal:**
-* [Test] Cross-reference the record count specified in arkivuttrekk with the actual record count in offentligjournal
-* [Test] Record created/finalized dates are within the extraction period start and end dates
+
+|Type|Results in|Title|Description|
+|---|---|---|---|
+|Test|Info,Warning|Registry entries|Provides information about the number of registry entries found in offentligJournal.xml and tests whether it matches the number of registry entries found in arkivstruktur.xml.|
+|Test|Info,Warning|Period containment of registry entries|Tests whether the created and finalized dates of all registry entries in offentligJournal.xml are within the archival period specified in arkivuttrekk.xml, and whether the same registry entries (same systemIDs) can be found in arkivstruktur.xml.|
+
+**addml:**
+
+|Type|Results in|Title|
+|---|---|---|
+|Test|Info,Error|Validity of the arkivstruktur.xml checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the arkivstruktur.xsd checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the loependeJournal.xml checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the loependeJournal.xsd checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the offentligJournal.xml checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the offentligJournal.xsd checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the endringslogg.xml checksum specified in arkivuttrekk.xml|
+|Test|Info,Error|Validity of the endringslogg.xsd checksum specified in arkivuttrekk.xml|
 
 **endringslogg:**
-* [Info] Number of changes in endringslogg
-* [Test] Number of system IDs found in endringlogg, but not in arkivstruktur
\ No newline at end of file
+
+|Type|Results in|Title|Description|
+|---|---|---|---|
+|Check|Info|Change log entries|Provides information about the number of change log entries in endringslogg.xml.|
+|Test|Info,Warning|Change log references|Tests whether all objects referenced in the change log can be found by their systemID in arkivstruktur.xml.|
+
+## Reports
+The tool can produce one or more reports containing validation results at the end of its execution (see the examples above for more information on how to request multiple reports). Currently supported reports are Excel (pre-2007 binary format), Excel (post-2007 OOXML format), and XML. All reports include
```

Comment:
`The tool can produce reports in different formats  (see the examples above for more information on how to request a report in multiple formats)`

`Currently supported formats are Excel...`